sqlite/sqlite.go

```diff
@@ -137,3 +137,18 @@ func OpenUnecryptedDB(path string) (*sql.DB, error) {
 
 	return db, nil
 }
+
+func ChangeEncryptionKey(path, key, newKey string) error {
+	db, err := openDB(path, key)
+
+	if err != nil {
+		return err
+	}
+
+	resetKeyString := fmt.Sprintf("PRAGMA rekey = '%s'", newKey)
```

Comment:
hey @shivekkhurana if a password manager is used to generate newKey which includes control chars, this could lead to a db crash (https://play.golang.org/p/rHetXaO9-9e). This function needs to handle/escape and to be tested for control chars in the newKey

Reply:
There is a sanitation process that fixes issues like the one you pointed. I checked with this password `111111""""""` and it worked fine. 

But this is a good point. I'm currently writing a validation function on the backend. It will implement a subset of password conditions pointed out by Corey above.